tests/features/steps/android_steps.rb

```diff
@@ -127,8 +127,6 @@
     And the payload field "events.0.device.totalMemory" is not null
     And the payload field "events.0.device.runtimeVersions.osBuild" is not null
     And the payload field "events.0.metaData.app.name" equals "MazeRunner"
-    And the payload field "events.0.metaData.app.packageName" equals "com.bugsnag.android.mazerunner"
-    And the payload field "events.0.metaData.app.versionName" is not null
```

Comment:
Should `versionName` assertion be dropped?

Reply:
I think that both `packageName` and `versionName` should no longer be automatically collected.

In the case of `versionName` this is because we already collect it with the key `app.version`. This has also caused a bit of confusion in the past about why we collect 3 separate keys: version/versionCode/versionName.